client/my-sites/marketplace/controller.js

```diff
@@ -24,11 +24,17 @@ export function renderThemesInstallPage( context, next ) {
 }
 
 export function renderMarketplaceThankYou( context, next ) {
-	const { plugins, themes } = context.query;
+	const { plugins, themes, onboarding } = context.query;
 	const pluginSlugs = plugins ? plugins.split( ',' ) : [];
 	const themeSlugs = themes ? themes.split( ',' ) : [];
 
-	context.primary = <MarketplaceThankYou pluginSlugs={ pluginSlugs } themeSlugs={ themeSlugs } />;
+	context.primary = (
+		<MarketplaceThankYou
+			pluginSlugs={ pluginSlugs }
+			themeSlugs={ themeSlugs }
+			onboardingFlow={ onboarding !== undefined }
```

Comment:
Nitpick, non-blocking: Since this is a boolean value, maybe we can change the prop to `isOnboardingFlow` so the intent is clearer.

Reply:
Fair, added at https://github.com/Automattic/wp-calypso/pull/81929/commits/2ad8c5ff6847efad7b8acc2ac9f610ddfee09e59